src/NzbDrone.Core/Tv/AddSeriesService.cs

```diff
@@ -67,7 +67,21 @@ public List<Series> AddSeries(List<Series> newSeries, bool ignoreErrors = false)
                     var series = AddSkyhookData(s);
                     series = SetPropertiesAndValidate(series);
                     series.Added = added;
-                    seriesToAdd.Add(series);
+                    if (!seriesToAdd.Any(f => f.TitleSlug == series.TitleSlug))
```

Comment:
This is only going to check against series added from this list, but this series could already exist in the DB and would fail for the same reason.

Reply:
@markus101 Doesn't SeriesTitleSlugValidator which is run from SetPropertiesAndValidate do that already?

Reply:
Yeah, you're right, still relevant for ID filtering after a redirect though.